package/harvester-os/files/usr/sbin/cos-installer

```diff
@@ -281,6 +281,9 @@ preload_rancherd_images()
 
 do_preload()
 {
+    cp $TARGET/etc/resolv.conf $TARGET/etc/resolv.conf.bak || true
+    cp /etc/resolv.conf $TARGET/etc/resolv.conf || true
```

Comment:
This looks awkward. Is the nameservers configuration not working at this stage?

Reply:
I doesn't work because we change root.

Reply:
Is bind-mount/umount a better way to have the resolv.conf ready?

Reply:
Fixed by [bind-mount.](https://github.com/harvester/harvester-installer/pull/93/commits/84794e1d338982cd10b2723dfe95b6bd784ee6ad#diff-6a84a5d8c064b0be0d59d2d26e27e2549fcf932c02014eb3f954bec7e780d7f7R155)